department-of-reuse/src/backend/RIndex.ts

```diff
@@ -22,6 +22,13 @@ export default class RIndex {
             }));
     }
 
+    async computeCitationCountForWork(sourceDoi: string) : Promise<number> {
```

Comment:
As this function technically has nothing to do with computing the RIndex, you might move it directly into GraphView.vue

Reply:
Will do!